test/onnx/export_onnx_tests_generator.py

```diff
@@ -52,13 +52,44 @@ def gen_module(testcase):
     return module
 
 
+def print_stats(FunctionalModule_nums, nn_module):
+    print("{} functional modules detected.".format(FunctionalModule_nums))
+    supported = []
+    unsupported = []
+    not_fully_supported = []
+    for key, value in nn_module.items():
+        if (value == 1):
+            supported.append(key)
+        elif (value == 2):
+            unsupported.append(key)
+        elif (value == 3):
+            not_fully_supported.append(key)
+
+    def fun(info, l):
+        print(info)
+        for v in l:
+            print(v)
+
+    for info, l in [["{} Fully Supported Operators:".format(len(supported)), supported],
```

Comment:
add some comments to make it clear: what is supported/semi-supported/unsupported?

Reply:
Sure, btw I updated it in quip.